t/translator.t

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env perl
+
+use v5.16;
+use warnings;
+use strict;
+use utf8;
+
+use Test::More;
+use Data::Dumper;
+
+###
+### Basic tests
+###
+
+BEGIN { use_ok 'Zonemaster::Backend::Translator'; }
+
+isa_ok 'Zonemaster::Backend::Translator', 'Zonemaster::Engine::Translator';
```

Comment:
I disagree. It’s customary to have a `BEGIN { use_ok 'SomeModule' }` in a unit test for a module. And the `isa_ok` assertion ensures that `Zonemaster::Backend::Translator` is a subclass of `Zonemaster::Engine::Translator`. These are all properties about the `Zonemaster::Backend::Translator` module I want to test, so I think it makes sense to keep them here.

Also, it isn’t clear what `t/00-load.t` is supposed to do. It seems like it is some kind of basic smoke test. It’s probably better left as it is.

Reply:
What I don't like about having `use_ok` is that it doesn't abort on failure, which IMHO is the reasonable thing to do. Instead it continues executing until it inevitably crashes and prints additional error output for no good reason. I admit this nothing more than a nuisance.

I understand `t/00-test.t` to be a simple smoke test that just verifies that all module can be loaded without error. If you have good test coverage this is redundant in a sense, but it's also fast and low effort to maintain. And this can be valuable if the rest of the test suite is slow.

With that I rest my case.